elm/README.md

```diff
@@ -20,3 +20,37 @@ In the private area, you'll see the contents for the logged in user and you can
 
 - add new contents with their tags;
 - filter the shown contents by tag.
+
+## Specification
+
+The `spec` folder contains some end-to-end acceptance tests written using [Quickstrom](https://quickstrom.io/).
+
+To run them, you could execute the following commands inside the `spec` folder, given your application is exposed on `localhost:8000`:
+
+```
+docker run --rm -d \
+  --name webdriver \
+  --network=host \
+  -v /dev/shm:/dev/shm \
+  -v $PWD:/spec \
+  selenium/standalone-chrome:3.141.59-20200826
+
+docker run --rm \
+  --network=host \
+  -v $PWD:/spec \
+  quickstrom/quickstrom \
+  quickstrom check \
+  --webdriver-host=webdriver \
+  --webdriver-path=/wd/hub \
+  --browser=chrome \
+  --reporter=html \
+  --html-report-directory=/spec/report \
+  --tests=10 \
+  --max-actions=50 \
+  --max-trailing-state-changes=1 \
+  --trailing-state-change-timeout=500 \
+  /spec/Tagger.spec.purs \
+  http://localhost:8000
```

Comment:
I am following these instructions, but running into:

```
2022-06-13 15:34:33 ERROR  Error Unable to connect to WebDriver server
```

Do you have any idea ?

Reply:
this happens if you don't have the `selenium/standalone-chrome` container running.

Scripting the two commands together should prevent this from happening